File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.py
class Solution:
    def reverseDegree(self, s: str) -> int:
        total = 0
        for i in range(0,len(s)):
            reversePos=ord('z')-ord(s[i])+1
            stringPos=i+1
            total+=reversePos*stringPos
        return total

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.cpp
class Solution {
public:
    int reverseDegree(string s) {
        int total = 0;
        for (int i=0; i<s.length();i++) 
        {
            int reversePos='z'-s[i]+1;
            int stringPos=i+1;
            total+=reversePos*stringPos;
        }
        return total;
    }
};